todoapp/codelab/25mar2017/04_ic_pin_disabled.md

```diff
@@ -0,0 +1,29 @@
+[\[Paso Anterior\]](03_activity_detail.md)
+
+#ic_pin_disabled.xml
+
+crearemos un nuevo vector drawable similar al que ya tenemos pero de color gris, usaremos este icono en los elementos de la lista que no tiene location. En la carpeta de drawable crea un nuevo archivo xml 
+
+<img src="http://i.imgur.com/GKryLQJ.png" >
+
+ llamalo `ic_pin_disabled.xml` y agrega el siguiente contenido:
+
+
+```xml
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
it is better to use the method that @Mathreyu used to create SVG drawables. Can we change this?, it is easier and more human friendly than... positions and xml.